### DISCUSSION TITLE
Anker Solix e1600 doesn't wake up from SLEEP mode (main user/power user in HA)

### DISCUSSION THREAD

**Discussion by dominicveit** [thomluther/ha-anker-solix#88] [Q&A] [answered]
Hello together,

I have successfully setup the Anker Automations in HA (with the main user). All works great until at night when e1600 goes into SLEEP mode when I switch "solarbank_e1600_erlaube_leistungsabgabe" OFF after a few minitues ("Ruhezustand"). And whatever I do I am not able to wake it up.

Any suggestions what I can do to wake up the e1600? Since I would like to run an automation based on sunset with -X offset.

Yesterday I had to manually get into my app with the main user and reset the schedule.

When I set the schedule with HA it also doesn't let the e1600 wake up.

Thanx for your help ans support. (However, since I am new, I don't know if I am doing anything wrong).

Cheers

Dom

**Comment by thomluther** [comment, marked as answer]
You probably hit a known issue as documented in the [INFO](https://github.com/thomluther/hacs-anker-solix/blob/main/INFO.md#schedule-service-details-and-limitations):

Situations have been observed during testing where an active home load export was applied in the schedule, but the solarbank did not react on the applied schedule. While the cloud schedule was showing the export enabled with a certain load set, verification in the Anker App presented a different picture: There the schedule load was set to 0 W in the slider which is typically not possible in the App...The only way to get out of such weird schedules on the appliance is to make the interval change via the Anker App since it may use other interfaces than just the cloud Api (Bluetooth and another Cloud MQTT server). This problem was only noticed when just a single resulting interval remains in the schedule that is sent via the Api, for example setting a new schedule or making an update with a full day interval. To avoid this problem, make sure your schedule has more than one time interval and you do NOT use the Set new Solarbank schedule service. Instead you can apply schedule changes via the Update Solarbank schedule service which is NOT covering the full day.

When you have such weird condition, the solarbank will not switch on anymore for discharge, since the Preset is 0W for any unknown reason. There is no way to get out of this condition with the Api.
You can only clear the schedule, then it should reset to the default behavior exporting 200 W until it is empty.
But every time when you send a new schedule interval in this condition, it will be a single slot that may be applied with a 0W output setting for unknown reason...
I avoid this problem by using > 1 interval in my daily schedule, and modifications are only done with the update schedule service for a single interval. I never use the set schedule service due to the observations above.

**Reply by dominicveit** [reply]
Hey @thomluther , thanx so much for your detailed feedback. Sorry that I didn't read that in the documentation. But this is great and detailed feedback, so thanx so much for that :)
I'll try this out. I have setup some multiple intervals. During the day I am only using the number set value for enable export and change of discharge value. I'll check how it is reacting. :)

**Comment by dominicveit** [comment]
Hey, I had a lot of flaws today... eveything sort of worked fine until in between BYPASS came in. I tried to keep this calm with an automation. However, the SLEEP mode again dropped in and I had today even deinstall and install again. So I am trying to watch with the guest user again, since I anyhow have a general energy usage of over 150W. But will check in my dashboard how the e1600 is behaving. Not sure if this is of any use, but maybe. As you said there might be different APIs inter connecting. Interstingly during "Überschuss" there's a totally fine behavior but then afterwards it's going to disturb all rules and acts weired.
Will watch into the behavior now in a more general way. But the misstion is to let it go on a rule basis without any interventiosn. But thats a long way to go I guess :) However, thanx for this really amazing work and that I can continue monitore :)

BTW... I am not a programmer but some knowledge left from like years ago... however, trying to keep up with the flow :)

**Reply by thomluther** [reply]
If you want to automate it, you may want to review[ this for automation ideas and concepts](https://github.com/thomluther/hacs-anker-solix/discussions/81).
I eliminated the sudden discharge during the day with the rules that I implemented there.

**Reply by dominicveit** [reply]
Hi @thomluther , thank you so much again for your tips and this phantastic article. I simplified it for my purposes since even during night time I have more energy consumption than 150W. I took a lot of ideas from you and will tweak it over the next days and weeks and of course when it goes into fall season and winter again. Also I didn't have the problem with the sleep mode anymore. Also I am not changing the schedule since it works very smooth with just using the parameters for allowing export and export amount. However, I'll likely add a schedule just for security reasons to be updated once a day soon. Really great integration and great documentation. I can imagine how much work this was. Thank you for your efforts. Cheers, Dom
